go.mod

```diff
@@ -1,19 +1,19 @@
 module github.com/IBM-Cloud/redli
 
-go 1.19
+go 1.21
```

Comment:
It would be great if you could upgrade go version to 1.23

Reply:
Done